editor/filesystem_dock.h

```diff
@@ -52,6 +52,7 @@
 #include "editor_dir_dialog.h"
 #include "editor_file_system.h"
 #include "script_create_dialog.h"
+#include "shader_create_dialog.h"
```

Comment:
The include should actually be in the `.cpp` file to avoid compilation dependency (other includes are wrong too).

Reply:
Fixed

Reply:
For others may be better to open a specified PR?

Reply:
Yeah, I noticed that lots of includes need a cleanup, so it could be left for another PR.